design-documents/graph-ql/coverage/product-reviews.graphqls

```diff
@@ -11,30 +11,33 @@ type ProductInterface {
     reviews(
         pageSize: Int = 20 @doc(description: "Specifies the maximum number of results to return at once."),
         currentPage: Int = 1 @doc(description: "Specifies which page of results to return."),
-    ): ProductReviews!
+        sort: ReviewSortInput @doc(description: "Specifies which field to sort on, and whether to return the results in ascending or descending order.")
+    ): ProductReviews! @doc(description: "ProductReview returns an array containing approved product reviews")
 }
 
 type ProductReviews {
     items: [ProductReview]! @doc(description: "An array of product reviews.")
     page_info: SearchResultPageInfo! @doc(description: "Metadata for pagination rendering.")
+    total_count: Int @doc(description: "The number of product reviews returned")
 }
 
 type ProductReview @doc(description: "Details of a product review") {
+    review_id: Int! @doc(description: "The ID assigned to the review")
     product: ProductInterface! @doc(description: "Contains details about the reviewed product")
-    summary: String! @doc(description: "The review summary (a.k.a title")
-    text: String! @doc(description: "The review text.")
+    title: String! @doc(description: "The review title")
+    review_text: String! @doc(description: "The review text.")
     nickname: String! @doc(description: "The customer's nickname. Defaults to customer name if logged in.")
     created_at: String! @doc(description: "Date indicating when the review was created.")
     average_rating: Float! @doc(description: "The average rating for product review.")
-    ratings_breakdown: [ProductReviewRating!]! @doc(description: "An array of ratings by rating category. For example quality, price.")
+    ratings: [ProductReviewRating!]! @doc(description: "An array of ratings by rating category. For example Quality, Price.")
 }
 
 type ProductReviewRating {
-    name: String! @doc(description: "The review rating name for example quality, price.")
-    value: String! @doc(description: "The rating value given by customer. Possible values by default: 1 to 5.")
+    name: String! @doc(description: "The review rating name. For example Quality, Price.")
+    percent: Float! @doc(description: "The rating percent given by customer to the review")
+    value: Int! @doc(description: "The rating value given by customer. Possible values by default: 1 to 5.")
```

Comment:
Based on https://github.com/magento/architecture/pull/343#discussion_r356165700 we should revert this change.